patterns/testimonials-two-columns-avatar.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Title: Two columns with avatar
+ * Slug: twentytwentyfive/testimonials-two-columns-avatar
+ * Keywords: testimonial
+ * Categories: testimonials
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|60"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignfull" style="padding-top:var(--wp--preset--spacing--60);padding-bottom:var(--wp--preset--spacing--60)"><!-- wp:columns {"align":"wide","style":{"spacing":{"blockGap":{"top":"var:preset|spacing|60","left":"var:preset|spacing|50"}}}} -->
+<div class="wp-block-columns alignwide"><!-- wp:column -->
+<div class="wp-block-column"><!-- wp:columns {"style":{"spacing":{"blockGap":{"top":"var:preset|spacing|40","left":"var:preset|spacing|40"}}}} -->
+<div class="wp-block-columns"><!-- wp:column {"width":"64px"} -->
+<div class="wp-block-column" style="flex-basis:64px"><!-- wp:image {"width":"64px","aspectRatio":"1","scale":"cover","sizeSlug":"large","linkDestination":"none","className":"is-style-rounded"} -->
+<figure class="wp-block-image size-large is-resized is-style-rounded"><img src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/image-from-rawpixel-id-5918459-original.webp" alt="<?php echo esc_attr_x( 'Picture of a person', 'Alt text for testimonial image', 'twentytwentyfive' ); ?>" style="aspect-ratio:1;object-fit:cover;width:64px"/></figure>
+<!-- /wp:image --></div>
+<!-- /wp:column -->
+
+<!-- wp:column -->
+<div class="wp-block-column"><!-- wp:quote {"className":"is-style-plain","style":{"typography":{"fontStyle":"normal","fontWeight":"400"},"spacing":{"blockGap":"var:preset|spacing|40"}},"fontSize":"large"} -->
+<blockquote class="wp-block-quote is-style-plain has-large-font-size" style="font-style:normal;font-weight:400"><!-- wp:group {"style":{"spacing":{"padding":{"top":"0","bottom":"0","left":"0","right":"0"},"margin":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained","justifyContent":"left","contentSize":"400px"}} -->
+<div class="wp-block-group" style="margin-top:0;margin-bottom:0;padding-top:0;padding-right:0;padding-bottom:0;padding-left:0"><!-- wp:paragraph {"style":{"typography":{"lineHeight":"1.1"}}} -->
```

Comment:
Similar to the other areas where I've taken this approach, the reason is to prevent long text lines at mid screen widths - this is more noticeable when longer text is used in a quote, see:

<img width="559" alt="image" src="https://github.com/user-attachments/assets/7a7c6ec4-20e7-4df6-9e23-2cb67f7756e5">

Happy to remove if you think it's not needed, but that was the reason I went for it 🙂

Reply:
Thanks for the clarification, @g-elwell. I see this is an edge case likely to happen on tablets in portrait mode, and your approach can solve it.

I'll defer to @beafialho so that she can give her input on what's best for the design.

Reply:
The decision will apply to these other comments:

- https://github.com/WordPress/twentytwentyfive/pull/118#discussion_r1755687401
- https://github.com/WordPress/twentytwentyfive/pull/118#discussion_r1755578274

Thanks ✨